### PR TITLE
feat: add endpoints

### PR DESCRIPTION
**Pull request by OropezaHugo**
here we add the endpoints for most of the requests

route:
http://localhost:5009/index.html

### CODE REVIEW

**Comment by AngheloZambrana** [comment]
More is a question what is the difference between return Ok(true); and return true;
In some controllers you use Ok(true) and others true, there is some reason for that, and its there no reason you can manage only one way to return.

File: backend/Controllers/UserController.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    {
+        return true;
+    }
+    
+    [HttpPut("{userId}")]
+    public ActionResult<bool> EditUserById(Guid userId, UserDTO userDto)
+    {
+        return true;
+    }
+}
```
Comment (AngheloZambrana) [inline backend/Controllers/UserController.cs]: Please add an endline in this class.

**Comment by OropezaHugo** [comment]
usando el Ok() se maneja explicitamente que la respuesta es code 200(exitosa), de esta forma nos permite tener claro que el contenido del Ok está...ok

**Review by JCarlosHidalgoSosa — APPROVED** [review]

File: backend/Controllers/ContactController.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    Guid.NewGuid()
+                }
+            }
+        });
+    }
+    
+    [HttpPost]
+    public ActionResult<bool> CreateContact([FromBody] ContactDTO contactDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/ContactController.cs]: Change to return Ok(true);

File: backend/Controllers/ContactController.cs [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    }
+    
+    [HttpPut("{contactId}")]
+    public ActionResult<bool> ChangeContact(Guid contactId, ContactDTO contactDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/ContactController.cs]: Change to return Ok(true);

File: backend/Controllers/HotelController.cs [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+                UserName = "Paco"
+            
+            }
+        });
+    }
+    
+    [HttpPost]
+    public ActionResult<bool> CreateHotel([FromBody] HotelDTO hotelDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/HotelController.cs]: Change to return Ok(true);

File: backend/Controllers/HotelController.cs [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    }
+    
+    [HttpPut("{hotelId}")]
+    public ActionResult<bool> UpdateHotel(Guid hotelId, HotelDTO hotelDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/HotelController.cs]: Change to return Ok(true);

File: backend/Controllers/ReservationController.cs [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                UserEmail = "test@test.com",
+                UserPhoneNumber = "12345678"
+            }
+        });
+    }
+    
+    [HttpPost]
+    public ActionResult<bool> CreateReservation(ReservationDTO reservationDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/ReservationController.cs]: Change to return Ok(true);

File: backend/Controllers/ReservationController.cs [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+    }
+    
+    [HttpPatch("{reservationId}")]
+    public ActionResult<bool> CancelReservation(Guid reservationId, bool Available)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/ReservationController.cs]: Change to return Ok(true);

File: backend/Controllers/ServiceController.cs [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+                ServiceID = Guid.NewGuid(),
+                Type = "Room Service"
+            }
+        });
+    }
+    
+    [HttpPost]
+    public ActionResult<bool> CreateService([FromBody] ServiceDTO serviceDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/ServiceController.cs]: Change to return Ok(true);

File: backend/Controllers/ServiceController.cs [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    }
+    
+    [HttpPatch("{serviceId}")]
+    public ActionResult<bool> ChangeServiceType(Guid serviceId, string type)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/ServiceController.cs]: Change to return Ok(true);

File: backend/Controllers/UserController.cs [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+                PhoneNumber = "0023456789",
+                UserID = Guid.NewGuid()
+            }
+        });
+    }
+    
+    [HttpPost]
+    public ActionResult<bool> CreateUser(UserDTO userDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/UserController.cs]: Change to return Ok(true);

File: backend/Controllers/UserController.cs [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    
+    [HttpDelete("{userId}")]
+    public ActionResult<bool> DeleteUserById(Guid userId)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/UserController.cs]: Change to return Ok(true);

File: backend/Controllers/UserController.cs [hunk 12]
```diff
@@ -0,0 +1,63 @@
...
+    }
+    
+    [HttpPut("{userId}")]
+    public ActionResult<bool> EditUserById(Guid userId, UserDTO userDto)
+    {
+        return true;
```
Comment (AngheloZambrana) [inline backend/Controllers/UserController.cs]: Change to return Ok(true);

**Review by AngheloZambrana — COMMENTED** [review]
Change to return Ok(true);

File: backend/Controllers/UserController.cs (hunk 1 above)
Comment (OropezaHugo) [reply backend/Controllers/UserController.cs]: done

File: backend/Controllers/ContactController.cs (hunk 2 above)
Comment (OropezaHugo) [reply backend/Controllers/ContactController.cs]: done

File: backend/Controllers/ContactController.cs (hunk 3 above)
Comment (OropezaHugo) [reply backend/Controllers/ContactController.cs]: done

File: backend/Controllers/HotelController.cs (hunk 4 above)
Comment (OropezaHugo) [reply backend/Controllers/HotelController.cs]: done

File: backend/Controllers/HotelController.cs (hunk 5 above)
Comment (OropezaHugo) [reply backend/Controllers/HotelController.cs]: done

File: backend/Controllers/ServiceController.cs (hunk 8 above)
Comment (OropezaHugo) [reply backend/Controllers/ServiceController.cs]: done

File: backend/Controllers/ServiceController.cs (hunk 9 above)
Comment (OropezaHugo) [reply backend/Controllers/ServiceController.cs]: done

File: backend/Controllers/UserController.cs (hunk 10 above)
Comment (OropezaHugo) [reply backend/Controllers/UserController.cs]: done

File: backend/Controllers/UserController.cs (hunk 11 above)
Comment (OropezaHugo) [reply backend/Controllers/UserController.cs]: done

File: backend/Controllers/UserController.cs (hunk 12 above)
Comment (OropezaHugo) [reply backend/Controllers/UserController.cs]: done

**Review by AngheloZambrana — APPROVED** [review]
Good job

**Pull request merged**